cdhweb/static_src/global/components/newsletter.scss

```diff
@@ -1,11 +1,11 @@
 /*
 Newsletter streamfield block.
 Note: this uses the same `includes/newsletter_form.html` rendered by in <footer>.
-We don't have a streamfield-specific template for it, therefore we're styling this one 
-the streamfield version based on its wagtail-generated class name `.block-newsletter`.
+We don't have a streamfield-specific template for it, but we conditionally add a 
+`.block--newsletter` wrapper when it's used in a streamfield context.
 */
 
-.block-newsletter {
+.block--newsletter {
```

Comment:
For the _most_ part I'm not styling these `.block--x` classes in terms of component styling. Mainly just for streamfield-wrapper spacing and alignment.

Newsletter is a special case because I want to keep the DOM light, as it's also used as a non-streamfield block in the footer.